v0/18-4-0_Custom_work_and_other_record_tabs_and_fields.markdown

```diff
@@ -1,8 +1,123 @@
+* Multi-select box - to select several options, hold the ctrl key down and select the options you need.
+* Plain text area - enter a short amount of free text.
+* Plain text box - enter longer amounts of free text
+* Rich text - enter free text here and add formatting as desired.
+* Check box - click on the box to fill in a tick
+* Radio button - click on the button to indicate a yes
+* Dates - Dates are entered in UK format by selecting from the day/month/year select boxes or using the popup calendar.
+* Date/Times - Dates and times are entered on a 24-hour clock from select boxes.
+* Date of birth selector - Dates are entered in UK format by selecting from the day/month/year select boxes and a current age is displayed
+* Number box - enter numbers only into a small text box.
+* Caption - a label is added, to separate fields within a tab.
+* Fixed text box - again this is descriptive text that cannot be altered in profiles, but can be used to add descriptive text or reminders about the other information on the tab
+* Year selector - choose the year from a drop-down (but not month or day)
+* Radio buttons - select one: add several radio buttons, only one of which can be selected
+* Checkboxes - select many: lets you select several options using tick boxes
+* Charity number lookup - enter a Charity Commission registered number and Lamplight will add a link to their page on the Charity Commission website
+* Twitter name: a text box that will let you look up their recent tweets
+* Web address - for additional websites - and will provide a link to the site
  
+__Adding fields to a system standard tab__
+
+All records have the tabs 'Where and Where', 'Attendance', ( this may be called 'respondent' or 'Involved' in some cases), and Details.  You can add additional fields to these tabs, by 'creating' them in System Admin.  Create a field with the correct name as below, (Capitalisation is essential), and add fields to it.  You will be able to come back and edit/remove these fields, but will not see any of the standard fields.  These can only be changed by Lamplight.  Use the appropriate, correctly capitalised name as below.
+
+![System Admin - adding fields to system tabs]({{imgpath}}1221a.png)
+
+_{Work}} records_
+When and where
+Attendance
+Details
+
+_{Outcome}} records_
+When and where
+Respondent (If you have this tab)
+
+_{Referral}} records_
+When and where
+Attendance (If you have this tab)
+Involving (If you have this tab)
+details
+
+_{{Grant}} records_
+When and where
+Attendance
+details
+
+
+__Adding custom columns to the 'Attendance' table__
+
+The other option you'll see in the 'Manage drop-down lists' section is 'view and edit custom columns on the attendance table'.  If you look at the {{Work}} records 'Attendance table in the attendance tab, you'll see that you have a number of columns, including name, attendance type, and role.  It's possible to add additional custom columns here, and you may already have some.
```

Comment:
If you look at the {{Work}} records 'Attendance table 
If you look at the {{work}} records Attendance table 

(capitalisation of work and the extra ')

Reply:
also capital W in next paras